certbot/account.py

```diff
@@ -270,9 +270,12 @@ def _save(self, account, acme, regr_only):
                 if hasattr(acme.directory, "new-authz"):
                     regr = RegistrationResourceWithNewAuthzrURI(
                         new_authzr_uri=acme.directory.new_authz,
-                        body=regr.body,
-                        uri=regr.uri,
-                        terms_of_service=regr.terms_of_service)
+                        body="",
```

Comment:
I used that because that's what we got back from the server when we queried new account with the only return existing argument, which seemed nicer to match.

Reply:
I don't think the server is sending the account object when we use `only_return_existing` though. [The spec](https://tools.ietf.org/html/draft-ietf-acme-acme-12#section-7.3.1) just says to return 200 with a location header and when I tested this, the body of the response is empty.

With this in mind, I still think we should use an empty dict here, but you can leave it as is if you disagree.